docs/man_pages/project/testing/publish.md

```diff
@@ -0,0 +1,43 @@
+publish
+==========
+
+<% if((isConsole && isMacOS) || isHtml) { %>
+Usage | Synopsis
+---|---
+General | `$ tns publish ios`
+
+Builds the project and publishes it to an Application Store.
+
+<% if(isHtml) { %>
+### Command Limitations
+
+* You can currently run `$ tns publish` only on OS X systems.
```

Comment:
This seems like an odd mix of two bullet points. Perhaps

* You can run `$ tns publish` only on OS X systems.
* Currently, you can run `$ tns publish` to publish only iOS applications.